main/parse.c

```diff
@@ -1803,6 +1862,12 @@ extern boolean parseFile (const char *const fileName)
 	{
 		if (Option.filter)
 			openTagFile ();
+
+#ifdef HAVE_ICONV
+		openConverter (EncodingMap && language <= EncodingMapMax ?
+				EncodingMap [language] : Option.encoding);
```

Comment:
shouldn't this also check whether `EncodingMap [language]` is `NULL`?
`(EncodingMap && language <= EncodingMapMax && EncodingMap [language]) ? EncodingMap [language] : Option.encoding`